client/src/index.js

```diff
@@ -4,7 +4,7 @@ import '@/styles/reset.css';
 import '@/styles/global.scss';
 import '@/assets/fonts/woowa-market-icons.css';
 
-// import TestPage from '@/pages/';
+// import TestPage from '@/pages/text-input-test-page.js';
```

Comment:
```suggestion
// import TestPage from '@/pages/';
```

다른 작업에서 충돌이 나지 않게 이것만 수정해주시면 될 것 같습니다.